ejb3/src/main/java/org/jboss/as/ejb3/deliveryactive/parser/EJBBoundMdbDeliveryMetaDataParser30.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.jboss.as.ejb3.deliveryactive.parser;
+
+import org.jboss.as.ejb3.deliveryactive.metadata.EJBBoundMdbDeliveryMetaData;
+import org.jboss.metadata.ejb.parser.jboss.ejb3.AbstractEJBBoundMetaDataParser;
+import org.jboss.metadata.property.PropertyReplacer;
+
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamReader;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Parser for EJBBoundMdbDeliveryMetaData components, namespace delivery:3.0
+ */
+public class EJBBoundMdbDeliveryMetaDataParser30 extends AbstractEJBBoundMetaDataParser<EJBBoundMdbDeliveryMetaData> {
```

Comment:
Rather than duplicate the parser, please consider parameterizing the existing one by namespace.
e.g. https://github.com/wildfly/wildfly/blob/main/ejb3/src/main/java/org/jboss/as/ejb3/timerservice/TimerServiceMetaDataParser.java